### DISCUSSION TITLE
User testimonials

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#327] [Show and tell]
We collected user statements about Freeplane in the old forum https://sourceforge.net/p/freeplane/discussion/758437/thread/5af6fd05 
and continue here. You can post your answers if you have not contributed yet. Add your own remarks if you like.

What do you use Freeplane for?

Which alternatives do you know?

Which advantages does Freeplane have?

What advantages do the alternatives have?

When do you use Freeplane and when do you use the alternatives?

**Comment by EdoFro** [comment]
hi Dimitry,
I think you should ask which Freeplane version are they using too.
Freeplane has improved quite a bit in the last versions, so if you ask about advantages it is good to know which version are they comparing to.

jm2c,
edo

**Comment by dpolivaev** [comment]
I moved this post to a new category "User testimonials" so that it can be unpinned
